avm/res/network/connection/main.json

```diff
@@ -5,8 +5,8 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.24.24.22086",
-      "templateHash": "14326202302024370679"
+      "version": "0.23.1.45101",
```

Comment:
Please undo. Would publish a new module version for now reason.

My recommendation - upgrade to the latest Bicep version ;)